.drone.yml

```diff
@@ -52,7 +52,7 @@ pipeline:
       - export DRONE_DIR="/drone"
       - git log | head -n 20
       - ./bin/add-coursier.sh
-      - ./bin/detect-community-build.sh
+      - ./bin/detect-community-build.sh && export RUN_COMMUNITY_BUILD=true
```

Comment:
`./bin/detect-community-build.sh && export RUN_COMMUNITY_BUILD=true || echo "Not running the community build."`

Otherwise this will kill the build for non-community-builds

Reply:
`"Not running the community build"` is already outputted by the script, but I'll make `|| 0` to avoid the failure.